### DISCUSSION TITLE
Equipo Hackaton

### DISCUSSION THREAD

**Discussion by Gabriand** [Launch-X-Latam/MisionFrontEnd#544] [Team Posts]
Banda ando buscando gente para nuestro equipo, quien esta interesado?

**Comment by Jpulido6** [comment]
Yo bro, aún no estoy en ningún equipo

**Comment by yamilmmoresco** [comment]
Si quieren tengo un grupo en discord

https://discord.gg/3Unq4PX4

**Comment by Gabriand** [comment]
Manden su usuario de teams

**Comment by yamilmmoresco** [comment]
lx08639@innovaccion.mx

**Comment by Josephr1318** [comment]
On Thu, 10 Nov 2022 at 1:20 PM Gabriand ***@***.***> wrote:

> Manden su usuario de teams
>
 Joseph Ruben Lopez (mail: ***@***.***)

**Comment by OrlandoHernandezGzz** [comment]
yo estoy interesado
lx15326@innovaccion.mx

**Comment by Gabriand** [comment]
Les envié mensaje al teams

**Comment by ShaneMS2098** [comment]
Si aún les queda una plaza para que me agreguen lx09872@innovaccion.mx

**Comment by JuanRodriguezDV** [comment]
lx08166@innovaccion.mx yo

**Comment by Jpulido6** [comment]
lx07496@innovaccion.mx

**Comment by MatiasBazan** [comment]
yo si todavía hay lugar: lx08634@innovaccion.mx

**Comment by Belem-Delfina** [comment]
Yo 
lx01004@innovaccion.mx

**Comment by AriadnaDalVqz** [comment]
yo lx17285@innovaccion.mx

**Comment by Sebastian-Longhitano** [comment]
Yo estoy interesado!

lx14356@innovaccion.mx

**Comment by JosueDuenas1995** [comment]
Aun hay espacio en tu equipo, lx06156@innovacion.mx

**Comment by paolocajo** [comment]
Estoy Interesado! lx12550@innovaccion.mx

**Comment by jlnunezobando** [comment]
Interesado lx08651@innovaccion.mx

**Comment by Gabriand** [comment]
Una disculpa a todos ya estamos completos, pero pueden armar sus grupos los que enviaron usuario

**Comment by mafer23** [comment]
https://discord.gg/QZa2UAHu

**Comment by renatocoello** [comment]
Hola buenas noches recién me estoy poniendo al corriente en las clases han sido unas semanas muy duras, aun hay puesto para entrar en algún grupo


este es mi usuario: ***@***.***
________________________________
De: Maria Fernanda Palencia ***@***.***>
Enviado: jueves, 10 de noviembre de 2022 18:31
Cc: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Equipo Hackaton (#79)


https://discord.gg/QZa2UAHu
